### PR TITLE
[Issue #2193] Update CODEOWNERS

### PR DESCRIPTION
**Pull request by coilysiren**
Relates to https://github.com/HHS/simpler-grants-gov/issues/2193

I updated CODEOWNERS to remove all the prior Nava folks, but haven't added new people yet. Mostly because I don't know where they go. I'll leave the next step to someone else, or maybe my future self.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # We are limiting this to 4 engineers and choosing not to select "Require review from Code Owners."
-* @acouch @aplybeah @chouinar @SammySteiner
+* @acouch @chouinar
 
-/documentation/ @sumiat @widal001 @andycochran @acouch @SammySteiner
-/frontend/ @andycochran @acouch @SammySteiner
-/api/ @jamesbursa @chouinar
-/analytics/ @ebuwa-evbuoma-fike @coilysiren @widal001 @AlexanderStephensonUSDS
+/documentation/ @sumiat @widal001 @andycochran @acouch
+/frontend/ @andycochran @acouch
```
Comment (acouch) [inline .github/CODEOWNERS:5]:
```suggestion
/frontend/ @andycochran @acouch @doug-s-nava 
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # We are limiting this to 4 engineers and choosing not to select "Require review from Code Owners."
-* @acouch @aplybeah @chouinar @SammySteiner
+* @acouch @chouinar
 
-/documentation/ @sumiat @widal001 @andycochran @acouch @SammySteiner
-/frontend/ @andycochran @acouch @SammySteiner
-/api/ @jamesbursa @chouinar
-/analytics/ @ebuwa-evbuoma-fike @coilysiren @widal001 @AlexanderStephensonUSDS
+/documentation/ @sumiat @widal001 @andycochran @acouch
```
Comment (acouch) [inline .github/CODEOWNERS:4]:
```suggestion
/documentation/ @mxk0  @widal001 @andycochran @acouch
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,11 +1,11 @@
 # We are limiting this to 4 engineers and choosing not to select "Require review from Code Owners."
-* @acouch @aplybeah @chouinar @SammySteiner
+* @acouch @chouinar
```
Comment (acouch) [inline .github/CODEOWNERS:2]:
```suggestion
* @acouch @chouinar @mdragon 
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 
-/documentation/ @sumiat @widal001 @andycochran @acouch @SammySteiner
-/frontend/ @andycochran @acouch @SammySteiner
-/api/ @jamesbursa @chouinar
-/analytics/ @ebuwa-evbuoma-fike @coilysiren @widal001 @AlexanderStephensonUSDS
+/documentation/ @sumiat @widal001 @andycochran @acouch
+/frontend/ @andycochran @acouch
+/api/ @chouinar
+/analytics/ @coilysiren @widal001 @AlexanderStephensonUSDS
```
Comment (acouch) [inline .github/CODEOWNERS:7]:
```suggestion
/analytics/ @coilysiren @widal001 @acouch 
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
 
-/infra/ @aplybeah @coilysiren
-/bin/ @aplybeah @coilysiren
-/.github/ @aplybeah @coilysiren
+/infra/ @coilysiren @acouch
```
Comment (acouch) [inline .github/CODEOWNERS:9]:
```suggestion
/infra/ @coilysiren @acouch @mdragon 
```

**Review by acouch — DISMISSED** [review]

**Comment by coilysiren** [comment]
🫡  @acouch updated!

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
-* @acouch @aplybeah @chouinar @SammySteiner
+* @acouch @chouinar @mdragon 
 
-/documentation/ @sumiat @widal001 @andycochran @acouch @SammySteiner
-/frontend/ @andycochran @acouch @SammySteiner
-/api/ @jamesbursa @chouinar
-/analytics/ @ebuwa-evbuoma-fike @coilysiren @widal001 @AlexanderStephensonUSDS
+/documentation/ @mxk0  @widal001 @andycochran @acouch
+/frontend/ @andycochran @acouch @doug-s-nava 
+/api/ @chouinar
```
Comment (mdragon) [inline .github/CODEOWNERS:6]: @coilysiren can you throw me on here, or I can if you're ok with me pushing a commit to your branch.

**Review by mdragon — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 6 above)
Comment (coilysiren) [reply .github/CODEOWNERS:6]: @mdragon yeah go for it!

**Review by acouch — APPROVED** [review]

**Comment by mdragon** [comment]
@coilysiren just flagging my change is made and I believe Aaron approved everything again

**Review by mdragon — APPROVED** [review]

**Pull request merged**